docs/guides/deploy.md

```diff
@@ -183,23 +186,23 @@ my_module.server.save("server.zip")
 
 The only noticeable difference between the deployment of modules and the deployment of circuits is that the methods `Client::encrypt`, `Client::decrypt` and `Server::run` must contain an extra `function_name` argument specifying the name of the targeted function.
 
-The encryption of an argument for the `inc` function of the module would be:
+For example, to encrypt an argument for the `inc` function of the module:
```

Comment:
has no verb

Reply:
These are fine without explicit verbs as they are understandable in the context and imply  "Here is how to...", we don't have to repeat "Here is how to..." 3 times. It's ok to omit part of the sentence especially when it's to give instructions in technical doc.

Reply:
I prefer sentences with verbs, myself, but ok